oauth2_provider/oauth2_validators.py

```diff
@@ -322,7 +322,7 @@ def invalidate_authorization_code(self, client_id, code, request, *args, **kwarg
 
         :raises: InvalidGrantError if the grant does not exist.
         """
-        deleted_grant_count = Grant.objects.filter(code=code, application=request.client).delete()
+        deleted_grant_count, _ = Grant.objects.filter(code=code, application=request.client).delete()
```

Comment:
if we're not going to use `_`, let's not assign it into scope.

Reply:
is deleted_grant_count the number of deleted records or the tuple when you do not specify the , _?

Reply:
https://docs.djangoproject.com/en/5.2/ref/models/querysets/#delete

>returns the number of objects deleted and a dictionary with the number of deletions per object type.

Reply:
okay, yeah, put the _ back. I've been working with TS a lot lately  and having trouble switching my brain back to python. mode. Sorry for the run around.

Reply:
Done